### ISSUE TITLE
Add options to Certificate Creation with plugin?

### ISSUE THREAD

**Issue by rbwrightjr91** [Netflix/lemur#2851]
This may be related (if tangentially) to #2178.

I'm writing a plugin for a third-party issuer (Sectigo), and we currently have two certificate types that can be created through them (in short a normal cert or a premium cert) and their api requires us to specify which type when requesting. 

Is there a way to add options to the certificate creation menu via plugin code? Or is this something I'd have to implement by altering source code? I'm sorry if this is documented, I couldn't find it anywhere.

**Comment by rbwrightjr91** [comment]
At the moment, I'm handling this with an issuer option that is selected when the authority is created, but I'd rather users only have to create one authority rather than multiple authorities (one for each cert type).

**Comment by hosseinsh** [comment]
hi Richard, 

this might be helpful to you: https://github.com/Netflix/lemur/pull/2841/files
Basically this addresses a similar case, where we wanted to support various types of profiles with an issuer. We then added new Authorities, all with the same issuer plugin, for instance in your case Sectigo, but different names. We then used the authority name, as a mapping to load the corresponding profile name from config file, as indicated in the test of the above PR.
